torch_em/data/datasets/light_microscopy/deepbacs.py

```diff
@@ -100,7 +100,7 @@ def get_deepbacs_data(path: Union[os.PathLike, str], bac_type: str, download: bo
     return data_folder
 
 
-def _get_paths(path, bac_type, split):
+def _get_deepbacs_paths(path, bac_type, split):
```

Comment:
If we make this a standard across dataset then we can remove the `_` and add docstrings.

Reply:
Thanks for the suggestion. I decided to go ahead and implement this for all microscopy datasets now.